### PR TITLE
Renamed package to old name for easier integration

### PR DESCRIPTION
**Pull request by popenta**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

_Replace this text by a clear and concise description of what this pull request is about and why it is needed. Be sure to explain the problem you're addressing and the solution you're proposing._
_For libraries, you can add a code sample of how to use it._
_For bugfixes, you can explain the previous behavior and how it was fixed._
_In case of visual features, please attach screenshots or video recordings to demonstrate the changes._

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/ledger-live-common/src/generated/specs.ts [hunk 1]
```diff
@@ -44,4 +43,5 @@ export default {
   tron,
   xrp,
   icon,
+  elrond,
```
Comment (andreibancioiu) [inline libs/ledger-live-common/src/generated/specs.ts:46]: I think the original sorting was alphabetical.

File: libs/ledger-live-common/src/families/elrond/bridge/mock.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    : new BigNumber(transaction.amount).plus(estimatedFees);
+  const amount = useAllAmount
+    ? account.balance.minus(estimatedFees)
+    : new BigNumber(transaction.amount);
+
+  if (amount.gt(0) && estimatedFees.times(10).gt(amount)) {
+    warnings.feeTooHigh = new FeeTooHigh();
+  }
+
+  // Fill up transaction errors...
```
Comment (andreibancioiu) [inline libs/ledger-live-common/src/families/elrond/bridge/mock.ts]: `...` can be omitted.

**Review by andreibancioiu — DISMISSED** [review]

File: libs/ledger-live-common/src/generated/specs.ts (hunk 1 above)
Comment (popenta) [reply libs/ledger-live-common/src/generated/specs.ts:46]: I don't think it was.

File: libs/ledger-live-common/src/families/elrond/bridge/mock.ts (hunk 2 above)
Comment (popenta) [reply libs/ledger-live-common/src/families/elrond/bridge/mock.ts]: done

**Review by AlexandruPislariu — APPROVED** [review]

**Review by AlexandruPislariu — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**